plugins/generic/pln/PLNPlugin.inc.php

```diff
@@ -434,7 +434,7 @@ function termsAgreed($journalId) {
 		$termsAgreed = unserialize($this->getSetting($journalId, 'terms_of_use_agreement'));
 		
 		foreach (array_keys($terms) as $term) {
-			if (!isset($termsAgreed[$term]) || ($termsAgreed[$term] == false)) return false;
+			if (!isset($termsAgreed[$term]) || ($termsAgreed[$term] == null)) return false;
```

Comment:
With ==, I don't this change has any effect -- null and false both evaluate to false.